### PR TITLE
Add ocsql project to contrib.go.opencensus.io

### PR DESCRIPTION
**Pull request by basvanbeek**
Add ocsql project to contrib.go.opencensus.io vanity urls to highlight the community supported status of this project.

See https://github.com/opencensus-integrations/ocsql/issues/16

### CODE REVIEW

**Review by odeke-em — APPROVED** [review]
LGTM, thank you @basvanbeek!

**Comment by odeke-em** [comment]
I'll defer to @rakyll as she has commit access and is the owner of this repo.

**Comment by odeke-em** [comment]
@bogdandrutu @rakyll kindly pinging y'all since a merge happened this morning :)

**Review by rakyll — APPROVED** [review]

**Comment by rakyll** [comment]
Can you resolve the conflict?

**Comment by basvanbeek** [comment]
conflict resolved.

**Comment by odeke-em** [comment]
@rakyll PTAL, @basvanbeek updated the conflict.

**Comment by odeke-em** [comment]
@songy23 or @rakyll please help us merge this.

**Pull request merged**
